layouts/partials/menu.html

```diff
@@ -7,25 +7,51 @@
           {{ range .Site.Menus.main }}
               {{ if .HasChildren }}
 
-            <li class="sub-menu{{if $currentNode.HasMenuCurrent "main" . }} active{{end}}">
-            <a href="javascript:;" class="">
-                {{ .Pre }}
-                <!--<i class="icon_desktop"></i>-->
-                <span>{{ .Name }}</span>
-                <span class="menu-arrow fa {{if $currentNode.HasMenuCurrent "main" . }}fa-angle-down{{else}}fa-angle-right{{end}}"></span>
-            </a>
-                <ul class="sub{{if $currentNode.HasMenuCurrent "main" . }} open{{end}}">
-                    {{ range .Children }}
-                    <li{{if $currentNode.IsMenuCurrent "main" . }} class="active"{{end}}><a href="{{.URL}}">{{ .Name }}</a> </li>
-                    {{ end }}
-                </ul>
-              {{end}}
+              <li class="sub-menu{{if $currentNode.HasMenuCurrent "main" . }} active{{end}}">
+                  {{ if .HasChildren }}
+                  <a href="javascript:;" class="">
+                      {{ .Pre }}
+		      <span>{{ .Name }}</span>
+                      <span class="menu-arrow fa {{if or ($currentNode.IsMenuCurrent "main" .) ($currentNode.HasMenuCurrent "main" .) }}fa-angle-down{{else}}fa-angle-right{{end}}"></span>
+                  </a>
+                  <ul class="sub nested-menu {{if or ($currentNode.IsMenuCurrent "main" .) ($currentNode.HasMenuCurrent "main" .) }} open {{end}}" style="{{if or ($currentNode.IsMenuCurrent "main" .) ($currentNode.HasMenuCurrent "main" .) }} display: block; {{else}}display: none; {{end}}">
+                      {{ range .Children }}
```

Comment:
Actually looks like theres a bunch of indentation issues in this file. I think it should be a tab of two spaces so example:
```
<div>
  <div>
    {{ if.whatever }}
    <ul>
    </ul>
  </div>
</div>
```